packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/utils/getActionTypesDefaultMessages.js

```diff
@@ -7,6 +7,9 @@ const actionTypes = {
   'media.create': 'Create media',
   'media.update': 'Update media',
   'media.delete': 'Delete media',
+  'media-folder.create': 'Create media folder',
+  'media-folder.update': 'Update media folder',
+  'media-folder.delete': 'Delete media folder',
```

Comment:
Just a question: why do you prefix these events with `media`? Shouldn't this follow the name of the plugin, which would be `upload` in this case?

Reply:
I wanted to stay consistent with the previously existing `media.create` events

Reply:
Where do these come from? Wondering because the BE refers to them as `files` and the FE as `assets` :) They don't have to be `media` files.

Reply:
It's coming from events being [dispatched](https://github.com/strapi/strapi/blob/c5dda8c9e91144835dbebd8e46ef985ea55dab91/packages/core/upload/server/services/upload.js#L70) in the event hub to power the webhooks. It matches the UI of the webhooks config in the admin as well

Reply:
Ok, thanks! :)